Feature/ltsp (#25)
Solvro/web-solvro-docs: src/content/docs/solvro/Strategy.mdx

---
title: Strategia KN Solvro
sidebar:
  order: 1
description: Długoterminowy plan strategiczny KN Solvro
---

import {
  Steps,
  Card,
  CardGrid,
  LinkCard,
  Aside,
} from "@astrojs/starlight/components";

> **<i>"Otwieramy drzwi do świata IT" ~ motto KN Solvro</i>**

## Początki

W 2018 roku grupa studentów napędzona kreatywnością i chęcią tworzenia założyła **Koło Naukowe Solvro**. Od tamtego momentu, przez te lata nasza organizacja podejmowała coraz to nowsze inicjatywy. Zarządy zmieniały się, a wraz z czasem początkowa wizja koła zanikała.

W 2024 roku, po objęciu władzy przez VII Zarząd, przeprowadzono restrukturyzację. Pół roku później, podczas wyjazdu do Lubawki, powstał zarys działań. Pod koniec roku opracowano dokument LTSP (Long Term Strategic Plan), który wytycza ścieżkę dla przyszłych zarządów, przekładając wiedzę obecnych członków na wartościowe wnioski zawarte w planie.

**KN Solvro** ma przed sobą świetlaną przyszłość, jednak to przyszli członkowie i zarządy będą odpowiadać za jej pielęgnowanie oraz dalszy rozwój 🚀.

## Nasze fundamenty

Wywodząc się z Politechniki Wrocławskiej, podążamy jej kierunkiem, jednocześnie definiując własną misję i wizję.

### Misja

> > **„Jesteśmy mostem między teorią i praktyką, kształcąc kolejne pokolenia

Reviewer: pytanie czy dawanie i markdown quotea i cudzysłowiów nie jest bezsensownym stylistycznym powtórzeniem

Reviewer 2: jest

  </Card>
  <Card title="M6" icon="heart">
    Liczba wydarzeń krajowych zorganizowanych przez KN Solvro
  </Card>
  <Card title="M7" icon="open-book">
    Liczba wydarzeń uczelnianych, w których uczestniczyło KN Solvro
  </Card>
  <Card title="M8" icon="approve-check-circle">
    Liczba nagród zdobytych przez KN Solvro
  </Card>
  <Card title="M9" icon="information">
    Liczba publikacji naukowych członków KN Solvro
  </Card>
  <Card title="M10" icon="puzzle">
    Liczba innowacyjnych rozwiązań wypracowanych przez KN Solvro
  </Card>
  <Card title="M11" icon="magnifier">
    Liczba informacji o KN Solvro, które ukazały się w mediach
  </Card>
  <Card title="M12" icon="comment-alt">
    Liczba warsztatów, szkoleń, kursów i innych działań edukacyjnych
    prowadzonych przez KN Solvro
  </Card>
  <Card title="M13" icon="random">
    Budżet KN Solvro, z naciskiem na środki finansowe i barter spoza
    Politechniki Wrocławskiej
  </Card>
  <Card title="M14" icon="github">
    Liczba członków aktywnie działająca w KN Solvro
  </Card>
  <Card title="M15" icon="linkedin">
    Liczba członków działająca w innych strukturach np. RUOS, projekty naukowe,
    samorządy itd.
  </Card>

>   <Card title="M16" icon="discord">

Reviewer: co to są do cholery za losowe ikkonki w tej sekcji wszystkie prawie xd

Author: Tylko takie są dostępne: https://starlight.astro.build/components/icons/

---
title: Strategia KN Solvro
sidebar:
  order: 1
description: Długoterminowy plan strategiczny KN Solvro
---

import {
  Steps,
  Card,
  CardGrid,
  LinkCard,
  Aside,
} from "@astrojs/starlight/components";

> > ### <i>"Otwieramy drzwi do świata IT" ~ motto KN Solvro</i>

Reviewer 2: co to za heading, nie widzę żeby on cokolwiek opisywał 

użyj pogrubienia zwykłego jeśli zależy Ci bardzo by to wstawić

Author: Zmieniłem na pogrubienie, ale głównie chodziło, żeby tekst  był większy i wyraźnie wyróżniony

## Początki

> W 2018 roku grupa studentów napędzona kreatywnością i chęcią tworzenia założyła **Koło Naukowe Solvro**. Od tego momentu, przez te lata nasza organizacja działała, podejmując coraz to nowsze inicjatywy. Zarządy się zmieniały, a wraz z nimi początkowa wizja zanikała.

Reviewer 3: ```suggestion
W 2018 roku grupa studentów napędzona kreatywnością i chęcią tworzenia założyła **Koło Naukowe Solvro**. Od tamtego momentu, nasza organizacja podejmowała coraz to nowsze inicjatywy. Zarządy zmieniały się, a wraz z czasem początkowa wizja koła zanikała.
```

Reviewer 3: Poprawa składni. Dodatkowo, myślę, że obarczenie upływu czasu zanikaniem wizji (koła) jest bardziej gustowne. Rozumiem, że w tym zdaniu o to chodziło, ale teraz mówimy to bezpośrednio.

Reviewer: Maybe "zmieniała"?

Reviewer 3: pravdive toto

> W 2024 roku, po objęciu władzy przez VII zarząd, nastąpiła restrukturyzacja. Pół roku później, w wyniku wyjazdu do Lubawki, powstał zarys działań. Pod koniec 2024 roku powstał ten dokument, czyli **LTSP** (Long Term Strategic Plan) mający na celu wytyczenie ścieżki dla kolejnych zarządów, przekładając wiedzę obecnego pokolenia członków na wartościowe wnioski zapisane w tym planie.

Reviewer 3: ```suggestion
W 2024 roku, po objęciu władzy przez VII zarząd, przeprowadzono restrukturyzację. Pół roku później, podczas wyjazdu do Lubawki, powstał zarys działań. Pod koniec roku opracowano dokument LTSP (Long Term Strategic Plan), który wytycza ścieżkę dla przyszłych zarządów, przekładając wiedzę obecnych członków na wartościowe wnioski zawarte w planie.
```

> **KN Solvro** ma przed sobą świetlaną przyszłość, ale to właśnie rolą kolejnych generacji członków i zarządów jest pielęgnowanie i rozwijanie tej przyszłości 🚀.

Reviewer 4: ```suggestion
**KN Solvro** ma przed sobą świetlaną przyszłość, jednak to przyszli członkowie i zarządy będą odpowiadać za jej pielęgnowanie oraz dalszy rozwój 🚀.
```

---

## Nasze fundamenty

> Wywodząc się z Politechniki Wrocławskiej utrzymujemy jej kierunek wyznaczając własną misję i wizję.

Reviewer 3: Może 'podążamy za jej kierunkiem'? Miałem problem ze zrozumieniem przekazu zdania, ale jestem ciekaw, co myślą inni.

### Misja

> > ##### „Jesteśmy mostem między teorią i praktyką, kształcąc programistów napędzających innowacje na Politechnice Wrocławskiej.”

Reviewer 3: Tak sobie myślałem, czy nie zmienić 'programistów' na '(kolejne) pokolenia'. Wiem, że sam tutaj dodawałem 'programistów', natomiast nasza działalność zaczęła już wykraczać poza ten label. No, będzie to też wymagało zmiany w paru innych miejscach xd

Reviewer 3: (ja też mogę być uwu)

Author: Hmm, ciekawe zagadnienie. Jak chcemy dać kolejne pokolenia [to tu coś musimy wstawić] 

Może kolejne pokolenia innowatorów, ale już mamy innowacje w słowa dalej
Kolejne pokolenia członków brzmi średnio
Kolejne pokolenia studentów?

> > ##### „Jesteśmy mostem między teorią i praktyką, kształcąc programistów napędzających innowacje na Politechnice Wrocławskiej.”

Reviewer 2: nie używaj headingów do tego!!! nie do tego są nagłówki, żeby pogrubiać tekst 😭

użyj zwykłego bolda

### Wizja

> > ##### „Wspieramy społeczność akademicką poprzez technologię, współpracę i wiedzę.”

Reviewer 2: nagłówek -> pogrubienie, do innych miejsc też to się aplikuje ale już nie chce mi się pisać 😭

### Wartości

> Adaptujemt wartości Politechniki Wrocławskiej nadając im znaczenie odpowiadające naszej działalności.

Reviewer 2: Adaptujemy

<Card title="Doskonałość" icon="starlight">

>   Dbając o jakość naszych rozwiązań używamy najnowszych technologii i staramy

Reviewer 3: staramy -> pozostajemy

</Card>
<Card title="Współpraca" icon="gitter">
  Ciągle współdziałamy z innymi kołami naukowymi, jednostkami Politechnicznymi i
  organizacjami, tworząc rozwiązania wspomagające naszą społeczność.
</Card>
<Card title="Otwartość" icon="seti:grails">
  Implementujemy innowację otwartą, gdzie większość naszego kodu jest open
  source, a nasze rozwiązania są publicznie dostępne.
</Card>

### Cele strategiczne

Obecnie nasze działania dyktuje **długoterminowy plan strategiczny**. Z misji, wizji i wartości wywodzą się cele strategiczne, taktyczne, a następnie operacyjne na dany okres.

<Card title="S1. Cyfryzacja życia studenckiego PWR" icon="laptop">
  Tworzymy rozwiązania od studentów dla studentów odgrywając kluczową rolę w
  informatyzacji Politechniki Wrocławskiej.
</Card>
<Card title="S2. Wspomaganie studentów w zakresie IT" icon="seti:graphql">
  Organizujemy warsztaty, wydarzenia oraz tworzymy zasoby edukacyjne, aby
  rozwijać kompetencje techniczne studentów.
</Card>
<Card title="S3. Stabilny wzrost i działanie KN Solvro" icon="rocket">
  Dbamy zarówno o wewnętrzne przekazywanie wiedzy, kompetencji i władzy jak i o
  zewnętrzny obraz KN Solvro.
</Card>

> Z celów strategicznych wywodzą się cele taktyczne i operacyjne na dany okrek, które kierują bieżącą działalności KN Solvro.

Reviewer 3: (żebym nie zapomniał tu wrócić)
Z celów strategicznych wywodzą się cele taktyczne i operacyjne na dany okres, które kierują bieżącą działalnością KN Solvro.

Reviewer 3: ```suggestion
Z celów strategicznych wywodzą się cele taktyczne i operacyjne na dany okres, które kierują bieżącą działalnością KN Solvro.
```

## Mierniki

Powyższe elementy wyznaczają kierunek działania. Zazwyczaj dopiero na poziomie celów operacyjnych i zadań pojawiają się konkretne rzeczy do realizacji.
Chcąc realnie ocenić postęp działalności KN Solvro wyróżniamy szereg mierników (KPI) odnoszących się do poszczególnych sfer.

<CardGrid>
  <Card title="M1" icon="laptop">
    Liczba użytkowników rozwiązań i aplikacji KN Solvro
  </Card>
  <Card title="M2" icon="document">
    Liczba wyświetleń materiałów promocyjnych KN Solvro
  </Card>
  <Card title="M3" icon="rocket">

>     Liczba wydarzeń międzynarodowych w których uczestniczyło KN Solvro

Reviewer 4: ```suggestion
    Liczba wydarzeń międzynarodowych, w których uczestniczyło KN Solvro
```

  </Card>
  <Card title="M4" icon="star">
    Liczba wydarzeń międzynarodowych zorganizowanych przez KN Solvro
  </Card>
  <Card title="M5" icon="list-format">

>     Liczba wydarzeń krajowych w których uczestniczyło KN Solvro

Reviewer 4: ```suggestion
    Liczba wydarzeń krajowych, w których uczestniczyło KN Solvro
```

  </Card>
  <Card title="M6" icon="heart">
    Liczba wydarzeń krajowych zorganizowanych przez KN Solvro
  </Card>
  <Card title="M7" icon="open-book">

>     Liczba wydarzeń uczelnianych w których uczestniczyło KN Solvro

Reviewer 4: ```suggestion
    Liczba wydarzeń uczelnianych, w których uczestniczyło KN Solvro
```

  </Card>
  <Card title="M8" icon="approve-check-circle">
    Liczba nagród zdobytych przez KN Solvro
  </Card>
  <Card title="M9" icon="information">
    Liczba publikacji naukowych członków KN Solvro
  </Card>
  <Card title="M10" icon="puzzle">
    Liczba innowacyjnych rozwiązań wypracowanych przez KN Solvro
  </Card>
  <Card title="M11" icon="magnifier">

>     Liczba informacji o KN Solvro, które pokazały się w mediach

Reviewer 4: pokazały -> ukazały ?

  </Card>
  <Card title="M12" icon="comment-alt">
    Liczba warsztatów, szkoleń, kursów i innych działań edukacyjnych
    prowadzonych przez KN Solvro
  </Card>
  <Card title="M13" icon="random">
    Budżet KN Solvro, z naciskiem na środki finansowe i barter spoza
    Politechniki Wrocławskiej
  </Card>
  <Card title="M14" icon="github">
    Liczba członków aktywnie działająca w KN Solvro
  </Card>
  <Card title="M15" icon="linkedin">
    Liczba członków działająca w innych strukturach np. RUOS, projekty naukowe,
    samorządy itd.
  </Card>
  <Card title="M16" icon="discord">
    Liczba partnerów z podziałem na wewnętrznych i zewnętrznych
  </Card>
</CardGrid>

## Realizacja strategii

<Steps>

> 1. W oparciu o cele strategiczne i wartości pod każdym z nich Zarząd definiuje cele taktyczne na daną kadencję.

Author: Chodziło, że pod każdym z celów strategicznych, są cele taktyczne, ale faktycznie średnio to brzmiało. Zmieniłem na "W oparciu o powyższe fundamenty zarząd definiuje plan działania na daną kadencję, który składa się m.in z celów taktycznych i kluczowych inicjatyw."

2. Przewodniczący Sekcji i Product Ownerzy definiują i odpowiadają za kwartalne cele operacyjne odnoszące się do poszczególnych celów i mierników.

3. Wynikające z nich zadania są realizowane przez aktywnych członków.

   <Aside type="caution">
     Pozostaie w statusie aktywnego członka KN Solvro wymaga wykonanie
     przynajmniej jednego zadania na semestr.
   </Aside>

4. Po kwartale rozliczane są cele operacyjne i tworzone jest krótkie podsumowanie działalności.

5. Pod koniec kadencji zarząd rozliczany jest z celów taktycznych i tworzone jest podsumowanie działalności.

</Steps>

<LinkCard
  title="Cele taktyczne VII Zarządu"

>   href=""

Reviewer 2: pusty link

>   description="Plan działania zdefiniowany przez VII zarząd na rok 2024/2025."

Author: VII Zarząd, bo to nazwa własna

...  i zarządy będą odpowiadać ... z małej, bo generalnie o nich piszę.

Poprawiłem całość wg. tego. Dzięki za uwagę

  Aside,
} from "@astrojs/starlight/components";

> <b>
>   <i>"Otwieramy drzwi do świata IT" ~ motto KN Solvro</i>
> </b>

## Początki

W 2018 roku grupa studentów napędzona kreatywnością i chęcią tworzenia założyła **Koło Naukowe Solvro**. Od tamtego momentu, przez te lata nasza organizacja podejmowała coraz to nowsze inicjatywy. Zarządy zmieniały się, a wraz z czasem początkowa wizja koła zanikała.

W 2024 roku, po objęciu władzy przez VII Zarząd, przeprowadzono restrukturyzację. Pół roku później, podczas wyjazdu do Lubawki, powstał zarys działań. Pod koniec roku opracowano dokument LTSP (Long Term Strategic Plan), który wytycza ścieżkę dla przyszłych zarządów, przekładając wiedzę obecnych członków na wartościowe wnioski zawarte w planie.

**KN Solvro** ma przed sobą świetlaną przyszłość, jednak to przyszli członkowie i zarządy będą odpowiadać za jej pielęgnowanie oraz dalszy rozwój 🚀.

## Nasze fundamenty

Wywodząc się z Politechniki Wrocławskiej, podążamy jej kierunkiem, jednocześnie definiując własną misję i wizję.

### Misja

> <b>
>   „Jesteśmy mostem między teorią i praktyką, kształcąc kolejne pokolenia
>   studentów napędzających innowacje na Politechnice Wrocławskiej.”

> > </b>

Reviewer 2: tu też markdown

### Wizja

> <b>
>   „Wspieramy społeczność akademicką poprzez technologię, współpracę i wiedzę.”

> > </b>

Reviewer 2: https://www.markdownguide.org/basic-syntax/#emphasis

jak możesz to używaj markdowna
